Test.cs

```diff
@@ -4,6 +4,8 @@ namespace FactorialCalculator
 {
     public class Factorial
     {
+        private branchName = "Dev"
```

Comment:
It is recommended to declare fields at the top of the class to maintain readability and organization. Also, consider marking the fields as readonly if they are not intended to be modified after initialization.